lisa/sut_orchestrator/azure/features.py

```diff
@@ -1629,51 +1629,17 @@ def get_hardware_disk_controller_type(self) -> Any:
         vm = get_vm(azure_platform, self._node)
         return vm.storage_profile.disk_controller_type
 
-    def get_raw_data_disks(self) -> List[str]:
-        # Handle BSD case
-        if isinstance(self._node.os, BSD):
-            return self._get_raw_data_disks_bsd()
-
-        # disk_controller_type == NVME
-        node_disk = self._node.features[Disk]
-        if node_disk.get_os_disk_controller_type() == schema.DiskControllerType.NVME:
-            # Getting OS disk nvme namespace and disk controller used by OS disk.
-            # Sample os_boot_partition:
-            # name: /dev/nvme0n1p15, disk: nvme, mount_point: /boot/efi, type: vfat
-            os_boot_partition = node_disk.get_os_boot_partition()
-            if os_boot_partition:
-                os_disk_namespace = get_matched_str(
-                    os_boot_partition.name,
-                    self.NVME_NAMESPACE_PATTERN,
-                )
-                os_disk_controller = get_matched_str(
-                    os_boot_partition.name,
-                    self.NVME_CONTROLLER_PATTERN,
-                )
-
-            # With NVMe disc controller type, all remote SCSI disks are connected to
-            # same NVMe controller. The same controller is used by OS disc.
-            # This loop collects all the SCSI remote disks except OS disk.
-            nvme = self._node.features[Nvme]
-            nvme_namespaces = nvme.get_namespaces()
-            disk_array = []
-            for name_space in nvme_namespaces:
-                if (
-                    name_space.startswith(os_disk_controller)
-                    and name_space != os_disk_namespace
-                ):
-                    disk_array.append(name_space)
-            return disk_array
-
-        # disk_controller_type == SCSI
+    def get_azure_scsi_discs(self) -> List[str]:
```

Comment:
`_get_scsi_disks`

Reply:
1. it's not used in other places, so it should add `_` to specify it's a private method.
2. all other places names are "disk", so keep it consistent to avoid using "disc".

Reply:
Some variables still using "disc", please change them to "disk".